package.json

```diff
@@ -41,12 +43,15 @@
     "oclif"
   ],
   "license": "MIT",
-  "main": "lib/index.js",
+  "main": "dist/index.js",
+  "dirname": "oex",
   "oclif": {
-    "commands": "./lib/commands",
     "bin": "oex",
+    "commands": "./dist/commands",
     "plugins": [
-      "@oclif/plugin-help"
+      "@oclif/plugin-autocomplete",
```

Comment:
Do we want to be opinionated about the plugins that people use? Or maybe it would suffice to add a list of useful plugins to the readme?

I ask because I think developers generally prefer to add functionality they want as opposed to removing functionality they don't want

Reply:
Yeah, you're right. I'll remove this.

Reply:
Maybe `@oclif/plugin-commands` might be a useful addition to the hello world`

Reply:
I agree with Mike, we should just make developers aware of those plugins but not include them in here so they don't have to remove them if they don't want them. Even if we think they're terribly useful.